README.md

```diff
@@ -14,12 +14,38 @@ Collar is a library which simplifies analytics debugging by showing events, scre
 
 ## Installation
 
+## CocoaPods
+
 Collar is available through [CocoaPods](https://cocoapods.org). To install it, simply add the following line to your Podfile:
 
 ```ruby
 pod 'Collar'
 ```
 
+## SwiftPM
+
+Adding the Collar as Swift package is possible in two ways: 
+
+* As a dependency to your app - using process integrated into Xcode UI
+* As a dependency to your existing Swift Package - modifying Package.swift file
+
+#### Via Xcode 
+
+To install Collar using the [Swift Package Manager](https://github.com/apple/swift-package-manager) you can follow the [tutorial published by Apple](https://developer.apple.com/documentation/xcode/adding_package_dependencies_to_your_app) using the URL for the Collar repo:
```

Comment:
As we discussed, I'm okay with this approach; still, if we just put 

```swift
 dependencies: [
     .package(url: "https://github.com/infinum/ios-collar.git")
 ]
 ```

without Xcode approach, I'm okay with that. To be consistent with other libs.